Woof/Woof/Model/Sitter/Sitter+Dummy.swift

```diff
@@ -20,5 +20,20 @@ extension Sitter {
             rating: 4.8,
             pricePerHour: 40.25
         )
+
+        /// The instance of dummy sitter list to use in preview of views.
+        static let sitters = [
+            // swiftlint:disable line_length
+            Sitter(id: UUID(), name: "Jane", surname: "Smith", phone: "0987654321", avatarUrl: Bundle.main.url(forResource: "EmilyDoeAvatar", withExtension: "jpg"), bio: "Certified dog trainer. Enjoys playing fetch and teaching new tricks.", rating: 4.8, pricePerHour: 20.0),
```

Comment:
Can we reformat the code:

```suggestion
            Sitter(
                id: UUID(),
                name: "Jane",
                surname: "Smith",
                phone: "0987654321",
                avatarUrl: Bundle.main.url(forResource: "EmilyDoeAvatar", withExtension: "jpg"),
                bio: "Certified dog trainer. Enjoys playing fetch and teaching new tricks.",
                rating: 4.8,
                pricePerHour: 20.0
            ),
```

so it's easier to read through the list of sitters?

Reply:
Done, fix with https://github.com/ios-course/ironfoudation-team-project/pull/48/commits/40c23a6da6f5db98503299f296ce96ec0b830a09